examples/acados/static_arm.py

```diff
@@ -89,14 +91,15 @@ def main():
     # --- Solve the program using ACADOS --- #
     ocp_acados = prepare_ocp(biorbd_model_path="models/arm26.bioMod", final_time=2, n_shooting=51, use_sx=True)
 
+    opts = SolverOptionsACADOS()
+    opts.nlp_solver_tol_comp = 1e-3
+    opts.nlp_solver_tol_eq = 1e-3
+    opts.nlp_solver_tol_stat = 1e-3
```

Comment:
Theoritically, this can be done directly in the constructor... Don't know which one I prefer though..
`opts = SolverOptionsACADOS(nlp_solver_tol_comp=1e-3, nlp_solver_tol_eq=1e-3, nlp_solver_tol_stat=1e-3)`
I let you decide

Reply:
That said, it feels that they refer to convergence (therefore should use the appropriate function)